src/home/home.html

```diff
@@ -5,8 +5,10 @@
         <img src="/nasa-pd4lo70LdbI-unsplash.png" />
       </div>
       <div class="title heading heading0">Seamlessly fundraise for your moonshot</div>
-      <div class="subtitle">PrimeLAUNCH is a platform for fair token launches in a
-        completely permissionless fashion.</div>
+      <div class="subtitle">
+        <div>PrimeLAUNCH is a platform for fair token</div>
```

Comment:
Just like in another PR, this is the wrong thing to do.  we should wrap text naturally.  The important thing is that the text wrap well at arbitrary page widths.  Do that by correctly setting the width of the container.

Reply:
Okay, I will leave the text to wrap naturally but would not set a width on the div because it would not produce the desired result of having the text wrap once on a particular word

Reply:
Usually it is fine as long as the width of the container is correct in the context of its surroundings. Splitting on a particular word is rarely important, and often not feasible at all browser page widths.